### ISSUE TITLE
Improve the wording of "Browse UTM Gallery" in the UI (it's not self-evident what "UTM Gallery" actually means)

### ISSUE THREAD

**Issue by rxhfcy** [utmapp/UTM#6194]
**Describe the issue**
The UI assumes that first-time users somehow automatically know that "UTM Gallery" actually means "a place where users can download prebuilt VM images".

This is what every brand new user sees after launching UTM for the very first time:
(Browse "**_UTM Gallery_**"?? Why? What is that?)

<img width="325" alt="What does UTM Gallery mean?" src="https://github.com/utmapp/UTM/assets/42669366/317513ae-961b-4959-9f74-1504268122c0">

Suggestion: replace "Browse UTM Gallery" with something more descriptive, such as:
- "Download a Prebuilt VM" (or "Virtual Machine")
- "Download a Ready-Made VM"
- "Download a VM Image"
- "Browse and Download VMs"
- "Explore VM Images"
- "Discover Ready-Made VMs"
- "Browse VM Catalog"
- "Browse VM Image Library"
- "Browse VM Image Gallery"
- "Download a VM image from UTM Gallery"

...or something like that.

PS. I think the **download** icon is an excellent choice here (it correctly implies that something will be downloaded)

**Comment by antonkratz** [comment]
"Explore VM Images" sounds as unclear to me as "Browse UTM Gallery". It makes me think I have a specific VM image, and I want to analyze (explore) it... maybe look what files are inside without running it? Or figure out some hardware parameters? "Download a pre-built Virtual Machine" seems clear.

**Comment by rxhfcy** [comment]
> "Download a pre-built Virtual Machine" seems clear.

Yes, something like that. I can't decide if "Virtual Machine" is better than "VM", maybe include both?

"Download a pre-built Virtual Machine (VM)"

Another data point: currently when the user clicks "+", the dialog has a link to "Download prebuilt from UTM Gallery..." which would IMHO already be much better than just the current "Browse UTM Gallery", but ideally "Virtual Machine" and/or "VM" should be explicitly spelled out because not all first-time novice users are mind readers? :)
